### DISCUSSION TITLE
CLI: Move requests are sent to eliminated Battlesnakes (even though the engine does not do this)

### DISCUSSION THREAD

**Discussion by Nettogrof** [BattlesnakeOfficial/feedback#33] [Bug Reports]
Currently the '/move' request are send to all snakes even those already eliminated.

CLI is often use for local developement, and snakes receving 'useless' /move request cause CPU usage.

On battlesnake.com,  '/move' request are only send to snakes still alive.

**Comment by andrewdmaclean** [comment]
Hey again @Nettogrof 👋 Just bumping this up the list for our dev team 😄

**Comment by bvanvugt** [comment]
We'd like to get this fixed in the next few weeks.

Issue here:
https://github.com/BattlesnakeOfficial/feedback/issues/64

**Comment by bvanvugt** [comment]
Should be fixed in [Rules v1.0.18](https://github.com/BattlesnakeOfficial/rules/releases/tag/v1.0.18)
